posthog/api/user.py

```diff
@@ -455,6 +458,30 @@ def redirect_to_site(request):
     return redirect("{}#__posthog={}".format(app_url, state))
 
 
+@authenticate_secondarily
+def redirect_to_website(request):
```

Comment:
Is there a reason you didn't use the existing redirect_to_site function for this? A lot of it looks similar / the same.

Reply:
The `redirect_to_site` function, is really different:
- generates a token using another way
- has a different query params
- uses permitted domains which are setup in the project settings (we only want posthog.com & localhost)
- redirects to another page